tests/integration_tests.rs

```diff
@@ -13,7 +13,7 @@ use std::time::SystemTime;
 use std::time::UNIX_EPOCH;
 use ureq;
 
-const WAIT_MS: u64 = 5;
+const WAIT_MS: u64 = 2;
```

Comment:
Nitpick: Such tricks make tests flaky by default. I don't have a better idea now, because I don't know this language that well, but I'll think about it.

The solution must be implemented in such a way, that if clock gets stuck for 1 sec, the test will wait and pass anyway.

Reply:
We probably shall brainstorm a better way of making the benchmark tests. I am not sure how to create something better at this point.

Reply:
Yes, I understand the issue, but I only mentioned the problem from testing point of view. I also said I don't have a better way for now, so it's a nitpick :)